domain/entity/latest_entry.go

```diff
@@ -0,0 +1,103 @@
+package entity
+
+import (
+	"fmt"
+	"strings"
+	"time"
+
+	"github.com/pkg/errors"
+	"google.golang.org/appengine/datastore"
+)
+
+const (
+	// LatestEntryCode* defines identify code
```

Comment:
**[golint]** <sub>reported by [reviewdog](https://github.com/haya14busa/reviewdog) :dog:</sub>
comment on exported const LatestEntryCodeTamai should be of the form "LatestEntryCodeTamai ..."